blocks/library/embed/index.js

```diff
@@ -22,6 +26,23 @@ registerBlock( 'core/embed', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
+		if ( ! url ) {
+			return (
+				<div className="blocks-embed is-placeholder">
```

Comment:
This could probably make use of the Slot Fill as well.  But I think that would be a future issue.